src/main/java/storage/format/TaskFormatter.java

```diff
@@ -0,0 +1,28 @@
+package com.leeyiyuan.storage.format;
+
+
+import com.leeyiyuan.task.Task;
+
+/**
+ * Represents a formatter to format a task.
+ */
+public abstract class TaskFormatter {
```

Comment:
This could be an interface.

But it's really subjective I guess...

Reply:
I had used an abstract class because I had expected to expand on the shared base functionalities of `TaskFormatters` in the future, in which case an abstract class makes more sense than an interface. At the moment, you are right, it is really subjective.